.github/dependabot.yaml

```diff
@@ -45,3 +45,27 @@ updates:
       action-deps:
         patterns:
           - '*'
+  - package-ecosystem: github-actions
+    directory: .github/actions/install-grafana
+    schedule:
+      interval: weekly
+    groups:
+      action-deps:
+        patterns:
+          - '*'
+  - package-ecosystem: github-actions
+    directory: .github/actions/install-prometheus
```

Comment:
Looks like the gen file updated this so that `install-prometheus` got specified twice?

Reply:
Odd. Good catch.